docs/README.md

```diff
@@ -3,12 +3,16 @@ The first step you want to do is create the python virtual environment:
 ```
 $ cd docs/
 $ python3 -m venv .venv
```

Comment:
```suggestion
$ python3 -m venv venv
```

This creates a virtualenv named `.venv`, let's have just `venv` instead. And we can either move this to the root of the repo (which is usually done in other projects, but here the venv is just for the docs), or modify our gitignore to include this `docs/venv/` directory so the virtualenv and it's entire contents don't appear in our version control.

Reply:
yeah, we don't need venv for entire repo, same for requirements.txt
I'll change venv command and gitignore

Reply:
Changed !